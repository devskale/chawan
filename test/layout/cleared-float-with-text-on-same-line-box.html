<!DOCTYPE html>
<div style="width: 40ch">
<div style="float: left; clear: both">float</div>
<div style="float: left; clear: left">float</div>
<div style="float: left">float</div>
test test test
<div style="float: right">float</div>
<div style="float: left; clear: left">float</div>
test test
</div>
