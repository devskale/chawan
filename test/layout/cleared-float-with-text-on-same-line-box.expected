floattest test test test test
floatfloat                         float
float
